src/api/auth/auth.controller.ts

```diff
@@ -10,12 +10,19 @@ export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
   @Post('sign-up')
-  async register(@Body() register: CreateUserDto) {
+  async register(@Body() register: CreateUserDto, hre: any) {
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'hre' is defined but never used.
<!-- __reviewdog__:ChBjNmYyYTYzZjQ0NGM5MDljEgZlc2xpbnQ= -->